pkgs/development/python-modules/torch-tb-profiler/default.nix

```diff
@@ -43,14 +42,16 @@ buildPythonPackage rec {
     "test_tensorboard_end2end"
     "test_tensorboard_with_path_prefix"
     "test_tensorboard_with_symlinks"
+    "test_autograd_api"
+    "test_profiler_api_with_record_shapes_memory_stack"
+    "test_profiler_api_without_record_shapes_memory_stack"
+    "test_profiler_api_without_step"
   ];
 
-  pythonImportsCheck = [ "torch_tb_profiler" ];
```

Comment:
we want to keep this

Reply:
build it yourself and test it out.

Reply:
`pythonImportsCheck` has now been reintroduced in line 39. Let's keep it here so as to not mess up the git blame.